vlrouter.go

```diff
@@ -769,6 +817,32 @@ func (self *Vlrouter) initFgraph() error {
 	})
 	bcastSrcFlow.Next(sw.DropAction())
 
+	// redirect dns requests to controller
+	macSaMask := net.HardwareAddr{0xFF, 0xFF, 0x00, 0x00, 0x00, 0x00}
+	macSa := net.HardwareAddr{0x02, 0x02, 0x00, 0x00, 0x00, 0x00}
```

Comment:
Is there a way to avoid hard coding the MAC signature here? If not, could we have a simple func with comments giving us this info?